CombatSprites.cs

```diff
@@ -37,48 +37,4 @@ public CombatSpriteColors SpriteColor
             set { spriteColor = value; }
         }
     }
-    public combatsprites(int color, int direction)

```

Comment:
so making a constructor in combat sprites doesnt help much
everything in here is great but I would put all of it in the projectile constructor, especially beccause there wont be any green shields